### PR TITLE
Fixes #20575: Port technique to 7.0 techniques API

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/20575

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: ansible-policies/packaging/postinst [hunk 1]
```diff
@@ -37,4 +37,4 @@ git commit -m "$CATEGORY installation"
 chmod +x $CONFIGURATION_PATH/rudder-synchronize $CONFIGURATION_PATH/ansible.sh $CONFIGURATION_PATH/remove_configuration
 
 # Import Techniques
-${CONFIGURATION_PATH}/rudder-synchronize import technique ${CONFIGURATION_PATH}/techniques
+curl --silent -k --header "Content-type: application/json" --header "X-API-Token: $(cat /var/rudder/run/api-token)" --request PUT https://localhost/rudder/api/latest/internal/techniques --data "@${CONFIGURATION_PATH}/techniques/ansible_policies.json"
```
Comment (ncharles) [inline ansible-policies/packaging/postinst:40]: are you sure of the "internal" here ?

**Comment by amousset** [comment]
PR updated with a new commit

File: ansible-policies/packaging/postinst (hunk 1 above)
Comment (amousset) [reply ansible-policies/packaging/postinst]: absolutely not

**Review by ncharles — APPROVED** [review]

**Pull request merged**